core/backend/src/test/ecsql/queries/AllOrAnyExp.ecsql.md

```diff
@@ -908,15 +908,15 @@ WHERE
 LIMIT 3
```

Comment:
Bah, yeah these two tests are nasty. we should move them to something that is schema independent. However, there is no 2 Minute drop in replacement for this that I could think of. We probably need to craft a test class with some elements and change the query to use it. Alternatively we could switch to statically typed values, something like:

```sql
SELECT
  *
FROM
  (VALUES (1), (2), (3)) AS myId(ECInstanceID)
WHERE
  ECInstanceID < ALL (
    SELECT
      ECClassId
    FROM
      (VALUES (3, 5), (2, 4)) AS t(ECClassId, RelECClassId)
  );
```

Expected rows: 1 with value 1
This is plain sqlite though, not sure if it's fully supported in ECSql.

Reply:
I think we can just check the result count with an outer query rather than matching the names everytime